### PR TITLE
set up sms & email templates

### PR DESCRIPTION
**Pull request by josh-chou**
https://github.com/SFDigitalServices/vaccination-distribution/issues/2

### CODE REVIEW

**Comment by josh-chou** [comment]
https://github.com/SFDigitalServices/vaccination-distribution/issues/1

**Comment by henryjiang-sfgov** [comment]
Unsubscribing currently doesn't go through this service.  It's managed on Sendgrid.

File: service/resources/notifications.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def send_notifications(self, notification_type, data):
 
         if user_email:
             subject = "Vaccination notificatioon"
```
Comment (cschlessinger) [inline service/resources/notifications.py:29]: We'll need to set the subject line up to change with the email content - can we put a TODO in here so we remember to delete this?

**Comment by cschlessinger** [comment]
> Unsubscribing currently doesn't go through this service. It's managed on Sendgrid.

@henryjiang-sfgov what about SMS opt-out? Are you saying the logic checks for not contacting unsubscribed users don't have to be handled here? What about the templates for the unsubscribe messages?

**Review by sashax — APPROVED** [review]
This is mergeable. If in fact Twilio and Sendgrid manage all the unsubscribe messages for us we can pull those files later.

**Pull request merged**